doc/users/prev_whats_new/github_stats_3.3.0.rst

```diff
@@ -254,7 +254,7 @@ Pull Requests (1066):
 * :ghpull:`17617`: Rewrite pdf test to use check_figures_equal.
 * :ghpull:`17654`: Small fixes to recent What's New
 * :ghpull:`17649`: MNT: make _setattr_cm more forgiving
-* :ghpull:`17644`: Doc 33 whats new consolidation
```

Comment:
The first two typo fixes are fine.

From here on, we only write out the titles of the pull requests. I prefer to keep them in sync with the PR title, even if that means, we have typos.

Reply:
Agree with Tim, we should not edit PR titles (other than _maybe_ re-syncing with what the titles of the PRs are now, but I really do not think that is worth the effort).

Reply:
I'll actually say the opposite and don't change these back; I'll fix the issues/PRs.

Reply:
ok